### DISCUSSION TITLE
Move the integrations in a subfolder and rename `binder`

### DISCUSSION THREAD

**Discussion by Sytten** [SeaQL/sea-query#663] [General]
Now with diesel we will have 4 integrations, they should probably be put in a subfolder.
I think we should consider deprecating `sea-query-binder` and renaming it to `sea-query-sqlx`.

**Comment by billy1624** [comment]
Hey @Sytten, I agreed, that would be more explicit and clean.

**Comment by tyt2y3** [comment]
I've published https://crates.io/crates/sea-query-sqlx

I don't think it's necessary to put them into sub-directory, I think it's managable now.
